modules/samples-operator-crd.adoc

```diff
@@ -4,25 +4,25 @@
 
 
 [id="samples-operator-crd{context}"]
-= Accessing the Samples Operator file
+= Accessing the Samples Operator configuration
 
 You can configure the Samples Operator by editing the file with the provided
 parameters.
 
 .Procedure
```

Comment:
I admit that it's out of scope, but you should have prereqs about installing oc and whatever permissions, if any, you need to run this command.